src/main/java/it/gov/pagopa/rtd/ms/rtdmsingestor/service/BlobRestConnector.java

```diff
@@ -63,6 +57,13 @@ public class BlobRestConnector {
   @Autowired
   StreamBridge sb;
 
+  @Autowired
+  IngestorRepository repository;
+
+  private int numNotEnrolledCards = 0;
```

Comment:
I use it on tests to ensure that the results match with the expected values 😬 but maybe it's a wrong choice 🤔

Reply:
it's ok, exposing some properties for observability purpose can be a way to test something. Just be careful to not use those value in production environment. For that i suggest to restrict the visibility of the getter methods, maybe using `protected` instead of `public`.